JamfSync/Model/SynchronizationProgress.swift

```diff
@@ -36,20 +36,21 @@ class SynchronizationProgress: ObservableObject {
     func updateFileTransferInfo(totalBytesTransferred: Int64, bytesTransferred: Int64) {
         if printToConsole {
             // NOTE: MainActor isn't called when processing the command line arguments since no UI is shown yet
-            setFileTransferVars(totalBytesTransferred: totalBytesTransferred, bytesTransferred: bytesTransferred)
+            setFileTransferVars(totalBytesTransferred: totalBytesTransferred, bytesTransferred: bytesTransferred, progress: self)
             if showProgressOnConsole {
                 printProgressToConsole()
             }
         } else {
             Task { @MainActor in
-                setFileTransferVars(totalBytesTransferred: totalBytesTransferred, bytesTransferred: bytesTransferred)
+                setFileTransferVars(totalBytesTransferred: totalBytesTransferred, bytesTransferred: bytesTransferred, progress: self)
             }
         }
     }
 
     func finalProgressValues(totalBytesTransferred: Int64, currentTotalSizeTransferred: Int64) {
         if printToConsole {
             // NOTE: MainActor isn't called when processing the command line arguments since no UI is shown yet
+            print("[SynchronizationProgress.finalProgressValues()] currentFileSizeTransferred: \(currentFileSizeTransferred ?? 0)   size: \(currentFile?.size ?? 0)")
```

Comment:
This should either be a debug log message or removed.

Reply:
removed